src/GitHub.VisualStudio/UI/Views/PullRequestCreationView.xaml.cs

```diff
@@ -14,6 +15,8 @@ public class GenericPullRequestCreationView : SimpleViewUserControl<IPullRequest
     [PartCreationPolicy(CreationPolicy.NonShared)]
     public partial class PullRequestCreationView : GenericPullRequestCreationView
     {
+        readonly Subject<ViewWithData> load = new Subject<ViewWithData>();
```

Comment:
Looks like this private member, `load` isn't used anywhere. Can it be removed or did you intend to use it?